src/incident_reports/docker-compose.yaml

```diff
@@ -1,5 +1,34 @@
 services:
+  mongo:
+    image: mongo:4.4.18
+    container_name: mongo_prod
+    ports: 
+      - 27017:27017  
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=mongo_db_user
+      - MONGO_INITDB_ROOT_PASSWORD=password
+    volumes:
+      - mongo:/data/db 
+      - ./configs/init_mongo_prod.js:/docker-entrypoint-initdb.d/init_mongo_prod.js
+    healthcheck:
+      test: ["CMD", "mongo", "--eval", "db.adminCommand('ping')"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+
   incident_reports:
     build: .
     ports:
       - 50005:8080
+    environment:
```

Comment:
Please add a "depends on" clause here so that the database is initialized first (since your server tries to connect as soon as it starts running)